static/sass/_pattern_p-docstring.scss

```diff
@@ -1,24 +1,134 @@
 @mixin p-charmhub-docstring {
-  .p-docstring {
-    @media (min-width: 1300px) {
-      .is-x-large-1 {
-        grid-column-end: span 1;
-      }
+  %bottom-section-padding--default {
+    padding-bottom: $spv-outer--regular-scaleable * 0.5;
 
-      .is-x-large-3 {
-        grid-column-end: span 3;
-      }
+    @media (min-width: $breakpoint-large) {
+      padding-bottom: $spv-outer--regular-scaleable;
+    }
+  }
+
+  %bottom-section-padding--shallow {
+    @media only screen and (max-width: $breakpoint-large) {
+      padding-bottom: $spv-outer--medium;
+    }
+
+    @media only screen and (min-width: $breakpoint-large) {
+      padding-bottom: $spv-outer--medium;
+    }
+  }
+
+  %bottom-section-padding--x-shallow {
+    @media only screen and (max-width: $breakpoint-large) {
+      padding-bottom: $spv-outer--small;
+    }
+
+    @media only screen and (min-width: $breakpoint-large) {
+      padding-bottom: $spv-outer--small;
+    }
+  }
+
+  .p-docstring__intro {
+    @extend %bottom-section-padding--shallow;
 
-      .is-x-large-4 {
-        grid-column-end: span 4;
+    overflow: auto; // prevent child margins overflowing
+  }
+
+  .p-docstring-block {
+    @extend %bottom-section-padding--default;
+
+    %vf-pseudo-border--top-light {
+      @extend %vf-pseudo-border--top;
+
+      position: relative;
+
+      &::after {
+        background-color: $colors--light-theme--border-low-contrast;
       }
+    }
+
+    & + & {
+      @extend %vf-pseudo-border--top; // do not add padding on nested blocks
+
+      margin-top: $spv-outer--shallow-scaleable;
+    }
+
+    & & {
+      @extend %bottom-section-padding--shallow;
+    }
+
+    & &:last-child {
+      padding-bottom: 0;
+    }
+
+    &__header {
+      @extend %bottom-section-padding--x-shallow;
+    }
+
+    // &__group:not(:last-child) {
```

Comment:
Is there any reason why this is commented out?

Reply:
no, deleted